src/surfaces/filtersRail/__test__/filtersRail.test.js

```diff
@@ -0,0 +1,113 @@
+/**
+ * To run this test from the church-management/client folder, execute the following command:
+ * npx jest ./src/surfaces/filtersRail/__test__/filtersRail.test.js
+ */
+
+import React from 'react';
+import {
+    BEM_NAME,
+    ROW_COLLAPSIBLE,
+    ROW_HEADING,
+    ROW_COMPONENTS,
+} from './constants';
+import mountWithTheme, { createMatchMedia } from '../../../testUtils/enzymeHelpers';
+import FiltersRail from '../filtersRail';
+
+describe('<FiltersRail />', () => {
+    const props = {
+        className: BEM_NAME,
+        id: BEM_NAME,
+        rows: [
+            {
+                collapsible: ROW_COLLAPSIBLE,
+                heading: ROW_HEADING,
+                components: ROW_COMPONENTS,
+            },
+        ],
+        isOpen: true,
+    };
+
+    it('Can render without problems', () => {
+        const wrapper = mountWithTheme(
+            <FiltersRail />,
+        );
+
+        expect(wrapper.exists()).toBe(true);
+    });
+
+    it('Should not render if breakpoint is below md.', () => {
+        window.matchMedia = createMatchMedia(375);
```

Comment:
Oh, cool!  This helps us simulate different viewport widths?

Reply:
Yep, kinda cool.